commands/create/partitions.go

```diff
@@ -52,7 +53,7 @@ func (c *partitions) run(_ *kingpin.ParseContext) error {
 
 	err = manager.CreatePartitions(c.topic, c.numberOfPartitions)
 	if err == nil {
-		fmt.Printf("The partitions of %s have been readjusted successfully.", c.topic)
+		c.logger.Logf(1, "The partitions of %s have been readjusted successfully.", c.topic)
```

Comment:
These are not logs. These are administrative command outputs which shouldn't be either controlled by logging verbosity level or prefixed with timestamp.